recipes/paraprobe/build.sh

```diff
@@ -0,0 +1,44 @@
+
+mpifort=$(which mpifort)
+mv $mpifort $mpifort.orig
+sed 's/\(enable_wrapper_rpath=\)"yes"/\1"no"/' $mpifort.orig >$mpifort
+chmod +x $mpifort
+
+echo ${SP_DIR}
+
+#test python building
+mkdir ${SP_DIR}/paraprobe_autoreporter
+cp -rf paraprobe-toolbox/code/paraprobe-autoreporter/src/python/* ${SP_DIR}/paraprobe_autoreporter
+
+mkdir ${SP_DIR}/paraprobe_parmsetup
+cp -rf paraprobe-toolbox/code/paraprobe-parmsetup/src/python/* ${SP_DIR}/paraprobe_parmsetup
+	
+mkdir ${SP_DIR}/paraprobe_transcoder
+cp -rf paraprobe-toolbox/code/paraprobe-transcoder/src/python/* ${SP_DIR}/paraprobe_transcoder
+
+#unzip voro++ and prepare
+cp thirdparty/mandatory/voroxx/voro++-0.4.6.tar.xz .
+tar xvf voro++-0.4.6.tar.xz
```

Comment:
If this package has to vendor this lib, you need to include its license.

Reply:
Thank you, how would that look like, would I simply add another file, for example:
```
about:
  license_file:
    - LICENSE
    - EXTRA_LICENSE
```

Reply:
Added the license file of Voro